assets/airports/eidw.json

```diff
@@ -875,136 +1072,578 @@
+                "34": ["BATED"]
+            },
+            "body": [],
+            "exitPoints": {
+                "LIFFY": ["KISHA", "LIFFY"]
+            },
+            "draw": [
+                ["KISHA"]
+            ]
+        },
+        "LIFFY": {
+            "icao": "LIFFY",
+            "name": "Liffy",
+            "suffix": {"10":"2E", "16":"5M", "28":"5A", "34":"5G"},
+            "rwy": {
+                "10": [["^DW551", "A30+"]],
+                "16": [["^DWS77", "A30+"]],
+                "28": [["^OE", "A30+"], "DUB"],
+                "34": [["^DWN26", "A30+"], "BATED"]
+            },
+            "body": [],
+            "exitPoints": {
+                "LIFFY": ["LIFFY"]
+            },
+            "draw": [
+                ["DW551", "LIFFY"],
+                ["DWN26", "_KISHA1", "_KISHA2", "_KISHA3", "_KISHA4", "_KISHA5", "_KISHA6", "_KISHA7", "_KISHA8", "_KISHA9", "_KISHA10", "_KISHA11", "_KISHA12", "_KISHA13", "_KISHA14", "_KISHA15", "_KISHA16", "_KISHA17", "_KISHA18", "_KISHA19", "_KISHA20", "_KISHA21", "_KISHA22", "_KISHA23", "_KISHA24", "_KISHA25", "_KISHA26", "_KISHA27", "BATED"], ["OE", "_LIFFY1", "_LIFFY2", "_LIFFY3", "_LIFFY4", "_LIFFY5", "_LIFFY6", "_LIFFY7", "_LIFFY8", "_LIFFY9", "_LIFFY10", "_LIFFY11", "_LIFFY12", "_LIFFY13", "_LIFFY14", "_LIFFY15", "_LIFFY16", "_LIFFY17", "_LIFFY18", "_LIFFY19", "_LIFFY20", "_LIFFY21", "_LIFFY22", "_LIFFY23", "_LIFFY24", "_LIFFY25", "_LIFFY26", "_LIFFY27", "_LIFFY28", "_LIFFY29", "_LIFFY30", "_DUB276", "DUB", "LIFFY"],
+                ["DWS77", "LIFFY"]
+            ]
+        },
+        "NEVRI": {
+          "icao": "NEVRI",
+            "name": "Nevri",
+            "suffix": {"10":"6E", "16":"5M", "28":"1A", "34":"1G"},
+            "rwy": {
+                "10": [["^DW551", "A30+"], "LEMTA"],
+                "16": [["^DWS77", "A30+"], "BATED", "ERUDA"],
+                "28": [["^OE", "A30+"], "DWN29"],
+                "34": [["^DWN26", "A30+"], "DWN29"]
+            },
+            "body": [],
+            "exitPoints": {
+                "NEVRI": ["NEVRI"]
+            },
+            "draw": [
+                ["DW551", "LEMTA", "NEVRI"],
+                ["DWS77", "_BATED1", "_BATED2", "_BATED3", "_BATED4", "_BATED5", "_BATED6", "_BATED7", "_BATED8", "_BATED9", "_BATED10", "_BATED11", "_BATED12", "_BATED13", "_BATED14", "_BATED15", "_BATED16", "_BATED17", "_BATED18", "_BATED19", "_BATED20", "_BATED21", "_BATED22", "_BATED23", "_BATED24", "_BATED25", "_BATED26", "_BATED27", "_BATED28", "_BATED29", "BATED", "ERUDA", "NEVRI"],
+                ["_KISHA6", "DWN29", "NEVRI"]
+            ]
+        },
+        "NEVRI(AB)": {
+            "icao": "NEVRI(AB)",
+            "name": "Nevri",
+            "suffix": {"10":"6F", "16":"5P", "28":"1B", "34":"1H"},
+            "rwy": {
+                "10": ["DW931", "LEMTA"],
+                "16": ["OP", "BATED", "ERUDA"],
+                "28": ["DUB", "DWN29"],
+                "34": ["DUB", "DWN29"]
```

Comment:
Are you sure? ERUDA Doesn't show up on the charts for NEVRI1H:
![eruda](https://user-images.githubusercontent.com/13125311/26904228-dce6e1d4-4bd8-11e7-904b-3f7915977d0b.PNG)